math/variations.11.cpp

```diff
@@ -0,0 +1,56 @@
+// Binomials
```

Comment:
According to the format, the first line is comment - a user-friendly name of the snippet. IMO "Binomials" is not a fitting description. Let's change it to "Variations with repetitions"